README.md

```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
 
 There is an issue where the libraries for font rendering are not properly bundled yet. You will need to install freetype.
 
-- Open an x86 terminal session with `arch -x86_64 zsh`
-- Install brew `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` (If you run into issues, let me know)
+
+### Setup your development and Homebrew environment
+- Ensure the Command Line Tools for Xcode 15 beta are installed. Visit https://developer.apple.com/downloads to download these tools.
+- Enter an x86_64 shell to continue the following steps in a Rosetta environment. All subsequent commands should be run within this shell.
+arch -x86_64 zsh
```

Comment:
Should be a separate list item/command and appropriately formatted as a code block.